pkgs/data/fonts/fira-code/symbols.nix

```diff
@@ -1,26 +1,36 @@
-{ stdenv, fetchzip }:
+{ stdenv, fira-code, python3, fontforge }:
 
-fetchzip {
-  name = "fira-code-symbols-20160811";
+let pythonEnv = python3.withPackages (p: [ p.fontforge ]);
```

Comment:
I honestly don't know whether this is best practice. This feels a little wrong to me to build a python environment for building an in-tree dependency...

Maybe something like this would work:
* add pythonFontForge as a dependency (set equal to python3Packages.fontforge in calling package)
* Add pythonFontForge to nativeBuildInputs
* check that the fontforge package is available in buildphase (something like ``python -m fontforge``)

Just guessing here.